context/app/static/js/shared-styles/sections/TableOfContents/TableOfContents.tsx

```diff
@@ -76,7 +76,11 @@ function ItemLinks({
         <ItemLink item={item} {...rest} />
         {subItems && (
           <IconButton size="small" onClick={() => setOpen((value) => !value)} color="primary">
-            {open ? <ExpandLess fontSize="small" /> : <ExpandMore fontSize="small" />}
+            {open ? (
+              <ExpandLess fontSize="small" aria-label="Collapse" />
+            ) : (
+              <ExpandMore fontSize="small" aria-label="Expand" />
```

Comment:
We could make these labels a bit more informative by including which section they expand/collapse